### PR TITLE
update predictionio formula to 0.8.5

### PR DESCRIPTION
**Pull request by pmariano**
Update predictionio formula from 0.7.3 to 0.8.5.

### CODE REVIEW

**Comment by bfontaine** [comment]
The build [failed](http://bot.brew.sh/job/Homebrew%20Pull%20Requests/20101/version=mountain_lion/testReport/junit/brew-test-bot/mountain_lion/audit_predictionio/) because you should use spaces instead of tabs for indentation.

File: Library/Formula/predictionio.rb [hunk 1]
```diff
@@ -2,33 +2,38 @@
...
+    Conf: #{libexec}/conf/pio-env.sh
+
+    Check status
+       pio status
+
+    For more details:
+      http://docs.prediction.io/install/install-linux/
     EOS
   end
+
```
Comment (jacknagel) [inline Library/Formula/predictionio.rb]: :scissors:

File: Library/Formula/predictionio.rb [hunk 2]
```diff
@@ -2,33 +2,38 @@
...
 
     libexec.install Dir['*']
-    bin.write_exec_script Dir["#{libexec}/bin/*"]
+    bin.write_exec_script libexec/"bin/pio"
 
-    mv "#{bin}/start-all.sh", "#{bin}/predictionio-start-all.sh"
-    mv "#{bin}/stop-all.sh", "#{bin}/predictionio-stop-all.sh"
+    inreplace libexec/"conf/pio-env.sh" do |f|
+       f.gsub! /#\s*ES_CONF_DIR=.+$/, "ES_CONF_DIR=#{HOMEBREW_PREFIX}/opt/elasticsearch/config"
+       f.gsub! /SPARK_HOME=.+$/, "SPARK_HOME=#{HOMEBREW_PREFIX}/opt/apache-spark"
```
Comment (jacknagel) [inline Library/Formula/predictionio.rb]: use `s` as the block parameter name, and use `Formula["formula name"].opt_prefix` rather than hard coding `opt`

**Comment by pmariano** [comment]
all suggestions were implemented.

thanks for the feedback

File: Library/Formula/predictionio.rb [hunk 3]
```diff
@@ -2,33 +2,37 @@
...
+    Be sure to have Elasticsearch and HBase running, and that the below conf reflects your current environment.
+
+    Conf: #{libexec}/conf/pio-env.sh
+
+    Check status
+       pio status
+
+    For more details:
+      http://docs.prediction.io/install/install-linux/
     EOS
```
Comment (mikemcquaid) [inline Library/Formula/predictionio.rb]: I'm not convinced these caveats are particularly helpful; I don't really understand them and I don't understand why they've changed from the previous ones. Maybe we should just remove them?

Comment (pmariano) [inline Library/Formula/predictionio.rb]: Remove caveats sounds good.

The previous version requires you to execute bash scripts (with no help command)  in order to start/pause predictionio. 

The new pio versions bring us a really better command line tool called `pio`. But as you suggest, caveats seems a bit unnecessary, the `pio help` should be enough.

**Pull request closed without merging**